lib/modules/activity/reply_card.dart

```diff
@@ -83,6 +84,15 @@ class ReplyCard extends StatelessWidget {
                       ),
                       const SizedBox(width: 10),
                     ],
+                    Visibility(
```

Comment:
I just realized I can't comment on unmodified lines of a PR 🙃
This concerns line 69: move `Consumer` outside of `if`. In its builder, provide the "More" button if the user of the reply is the viewer and the "Reply" button otherwise. Since `Consumer` provides its own `ref`, you can remove the other one from the `ReplyCard` constructor.